eg/Classes/AutostartItem.py

```diff
@@ -20,6 +20,30 @@
 import eg
 from MacroItem import MacroItem
 from TreeItem import HINT_MOVE_INSIDE, HINT_MOVE_AFTER
+from datetime import datetime
+
+
+class Time(object):
+    _datetime = None
+
+    def __init__(self):
+        self._datetime = datetime.now()
+
+    def __getattr__(self, item):
+        return getattr(self._datetime, item)
+
+    def __str__(self):
+        t_fmt = '%A, %B %d{}, %Y @ %H:%M:%S %p'
+        if 4 <= self.day <= 20 or 24 <= self.day <= 30:
+            t_fmt = t_fmt.format('th')
+        else:
+            t_fmt = t_fmt.format(
+                ["st", "nd", "rd"][t.tm_mday % 10 - 1]
```

Comment:
`NameError: global name 't' is not define`

Reply:
could be let me look at the code. i could have messed it up somehow. tho i did test it and it worked. maybe i missed the error when it happened.